doc-content/enterprise-only/admin-and-config/business-central-settings-security-policy-con.adoc

```diff
@@ -0,0 +1,24 @@
+[id='business-central-settings-security-policy-con']
+= Security policy
+
+Every application contains a single security policy which is used every time the system checks a permission. The set of permissions assigned to every role and/or group is called the security policy.
```

Comment:
Consider revising: A security policy is a set of permissions that are assigned to specific roles or groups. Each application contains one security policy that is used each time the system checks a permission for a role or group.